test/unit/attributes.js

```diff
@@ -451,7 +451,7 @@ QUnit.test( "attr(String, Object)", function( assert ) {
 
 	$radio = jQuery( "<input>", {
 		"value": "sup",
-		"type": "radio"
+		"TYPE": "radio"
```

Comment:
Yea, lowercase is covered elsewhere. I wanted to have one place where we ensure we still use the `type` attrHook even when the name passed is not lowercase. I'll add a comment.